packages/boilerplate-generator/boilerplate_web_template.js

```diff
@@ -0,0 +1,90 @@
+// Template function for rendering the boilerplate html
+// Replicates the template defined in boilerplate_web.browser.html
+// XXX Does not necessarily preserve formatting (e.g. additionalStaticJs newlines)
+// Arguments: root : { htmlAttributes, css : [{ url }], bundledJsCssUrlRewriteHook : Function, head, dynamicHead, body, dynamicBody, inlineScriptsAllowed, additionalStaticJs, meteorRuntimeConfig }
+
+Boilerplate_Web_Browser_Template = function(root) {
```

Comment:
This isn't our normal variable naming style.  I think BoilerplateWebBrowserTemplate, or even just WebBrowserTemplate since we're not exporting it out of the package.